packages/cardpay-cli/index.ts

```diff
@@ -68,6 +77,47 @@ const { network, mnemonic = process.env.MNEMONIC_PHRASE, tokenAddress, amount, a
       command = 'prepaidCardCreate';
     }
   )
+  .command(
+    'usd-price <token> <amount>',
+    'Get the USD value for the USD value for the specified token in the specified amount',
+    (yargs) => {
+      yargs.positional('token', {
+        type: 'string',
+        description: 'The token symbol (without the .CPXD suffix)',
+      });
+      yargs.positional('amount', {
+        type: 'string',
+        description: 'The amount of the specified token (*not* in units of wei)',
+      });
```

Comment:
It might be convenient to have `amount` default to `1`

Reply:
Actually we'd want to default to 10<sup>18</sup>, since everything is in units of wei. but yeah, that is a good suggestion

Reply:
or wait--duh, from teh CLI we use non-wei, it's the SDK that uses wei--nevermind